.env.example

```
@@ -22,27 +22,33 @@ JWT_AUDIENCE=Web
+# Email Settings
 EMAILSETTINGS__HOST=smtp.gmail.com
 EMAILSETTINGS__PORT=587
 EMAILSETTINGS__FROM=
 EMAILSETTINGS__USERNAME=
-EMAILSETTINGS__PASSWORD=""
+EMAILSETTINGS__PASSWORD=
 
+# Cloudinary
 CLOUDINARY__CLOUDNAME=
 CLOUDINARY__APIKEY=
+<<<<<<< HEAD
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The <<<<<<< HEAD key should be with a value or have an equal sign

(KeyWithoutValue)